src/server.py

```diff
@@ -33,10 +26,6 @@ def listen_to(server):
         try:
             body, content_type = resolve_uri(parse_request(message))
             conn.sendall(response_ok() + content_type + b'\r\n\r\n' + body + b'\r\n\r\n')
```

Comment:
Imagine if you had this line of code here on 28 instead:

```python
response = response_ok(body, content_type)
```
Then below in lines 30-38 you could build bad responses like so:
```python
except AttributeError:
    response = response_error(405, u'Method Not Allowed')
...
```
And then finally, in the `finally` clause on lines 39/40 you could do this:
```python
finally:
    conn.sendall(response)
    conn.close()
```
This would reduce the number of locations in your code that dealt with sending bytes over the wire to exactly one.  That's a bit less repetitious.  And easier to debug if that one line is causing the problem.